### PR TITLE
docs: update Day.js doc

### PR DESCRIPTION
**Pull request by iamkun**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: docs/react/use-custom-date-library.zh-CN.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 order: 7.5
-title: 替换 Moment.js
+title: 使用自定义日期库
 ---
 
-你可以用自定义日期库（[day.js](https://day.js.org)、[date-fns](https://date-fns.org)）替换 Moment 以优化打包大小。在这里我们提供了两种方式来实现替换:
+你可以用自定义日期库如（[moment](http://momentjs.com/)、[date-fns](https://date-fns.org)）。在这里我们提供了两种方式来实现替换:
```
Comment (zombieJ) [inline docs/react/use-custom-date-library.zh-CN.md:6]: 加个 Why DayJS 然后夸夸 DayJS ~~~

File: docs/react/use-custom-date-library.zh-CN.md [hunk 2]
```diff
@@ -82,33 +82,27 @@ export { default as TimePicker } from './TimePicker';
...
 
 ```js
 // webpack-config.js
-import AntdDayjsWebpackPlugin from 'antd-dayjs-webpack-plugin';
+import AntdMomentWebpackPlugin from 'antd-moment-webpack-plugin';
 
 module.exports = {
   // ...
-  plugins: [new AntdDayjsWebpackPlugin()],
+  plugins: [new AntdMomentWebpackPlugin()],
```
Comment (afc163) [inline docs/react/use-custom-date-library.zh-CN.md:105]: 业务中测试过么？

File: docs/react/use-custom-date-library.zh-CN.md [hunk 3]
```diff
@@ -82,33 +82,27 @@ export { default as TimePicker } from './TimePicker';
...
-
-## antd-dayjs-webpack-plugin
-
-我们还提供另一种实现方式。使用 `antd-dayjs-webpack-plugin` 插件，无需对现有代码做任何修改直接替换成 `Day.js`。请参考 [antd-dayjs-webpack-plugin](https://github.com/ant-design/antd-dayjs-webpack-plugin)。
+我们还提供另一种实现方式。使用 `antd-moment-webpack-plugin` 插件，无需对现有代码做任何修改直接替换成 `Moment.js`。请参考 [antd-moment-webpack-plugin](https://github.com/ant-design/antd-moment-webpack-plugin)。
 
 ```js
 // webpack-config.js
-import AntdDayjsWebpackPlugin from 'antd-dayjs-webpack-plugin';
+import AntdMomentWebpackPlugin from 'antd-moment-webpack-plugin';
```
Comment (afc163) [inline docs/react/use-custom-date-library.zh-CN.md:101]: 这个包放 @ant-design scope 下吧。

File: docs/react/use-custom-date-library.zh-CN.md (hunk 2 above)
Comment (iamkun) [reply docs/react/use-custom-date-library.zh-CN.md:105]: 嗯 用 next npm 包测过了

File: docs/react/use-custom-date-library.zh-CN.md (hunk 3 above)
Comment (afc163) [reply docs/react/use-custom-date-library.zh-CN.md:101]: @ant-design/dayjs-webpack-plugin 也可以搞个

File: docs/react/faq.zh-CN.md [hunk 4]
```diff
@@ -35,9 +35,9 @@ title: FAQ
 
 你可以覆盖它们的样式，但是我们不推荐这么做。antd 是一系列 React 组件，但同样是一套设计规范。
 
-## 如何使用 Day.js 替换 Moment.js 来减小打包大小？
+## 如何使用 Day.js 替换 Day.js 来减小打包大小？
```
Comment (dancerphil) [inline docs/react/faq.zh-CN.md:38]: `用 Day.js 替换 Day.js`

**Pull request merged**